app/views/posts/index.html.erb

```diff
@@ -1 +1,6 @@
-<h1> Displays all posts from a user </h1>
\ No newline at end of file
+<div id="posts">
+<%= render @user %>
+<% @posts.each do |post| %>
+<%= render post %>
```

Comment:
- [ ] In the post index, User should see the  5 most recent comments. Use `five_most_recent_comments` method to show recent comments.